scipy/stats/_continuous_distns.py

```diff
@@ -3582,56 +3583,943 @@ class levy_stable_gen(rv_continuous):
 
     Notes
     -----
-    Levy-stable distribution (only random variates available -- ignore other
-    docs)
+    ..math:: \alpha-stable  distributions form a rich family of probability distributions 
```

Comment:
For online math to render, it needs backticks: `:math:backtick \alpha backtick`.

Note that `.. math::` typesets display math.